packages/runtime/test/dataViews/requestItems/FormFieldRequestItemDVO.test.ts

```diff
@@ -0,0 +1,410 @@
+import { AcceptFormFieldRequestItemParametersJSON, DecideRequestItemParametersJSON } from "@nmshd/consumption";
+import { FormFieldRequestItem, SelectionFormFieldSettings, SelectionFormFieldSettingsJSON, StringFormFieldSettings, StringFormFieldSettingsJSON } from "@nmshd/content";
+import {
+    ConsumptionServices,
+    CreateOutgoingRequestRequest,
+    DataViewExpander,
+    FormFieldAcceptResponseItemDVO,
+    FormFieldRequestItemDVO,
+    IncomingRequestStatusChangedEvent,
+    LocalRequestStatus,
+    OutgoingRequestStatusChangedEvent,
+    RequestMessageDVO,
+    TransportServices
+} from "../../../src";
+import {
+    establishRelationship,
+    exchangeAndAcceptRequestByMessage,
+    exchangeMessageWithRequest,
+    MockEventBus,
+    RuntimeServiceProvider,
+    sendMessageWithRequest,
+    syncUntilHasMessageWithRequest,
+    syncUntilHasMessageWithResponse,
+    TestRuntimeServices
+} from "../../lib";
+
+const serviceProvider = new RuntimeServiceProvider();
+let runtimeServices1: TestRuntimeServices;
+let runtimeServices2: TestRuntimeServices;
+let transportServices1: TransportServices;
+let transportServices2: TransportServices;
+let expander1: DataViewExpander;
+let expander2: DataViewExpander;
+let consumptionServices2: ConsumptionServices;
+let eventBus1: MockEventBus;
+let eventBus2: MockEventBus;
+let address2: string;
+
+let requestContent: CreateOutgoingRequestRequest;
+let responseItems: DecideRequestItemParametersJSON[];
+
+afterAll(() => serviceProvider.stop());
+
+beforeEach(function () {
+    eventBus1.reset();
+    eventBus2.reset();
+});
+
+describe("FormFieldRequestItemDVO with StringFormFieldSettings", () => {
+    const aMin = 1;
+    const aMax = 100;
```

Comment:
same as in the other test file. If you want to use these constants please move them to every test to avoid jumping back and forth.